runtime/lua/vim/lsp/util.lua

```diff
@@ -444,6 +444,21 @@ function M.jump_to_location(location)
   return true
 end
 
+-- Show a LocationLink or Location in a floating windows
+function M.preview_location(location)
```

Comment:
why is this called preview instead of peek? "preview window" in N/Vim has a specific meaning. Maybe "popup" is better, or else "peek".

I see `open_floating_preview` exists already, that should be renamed...

Reply:
It used to be called `peek_location` -- then I renamed it because `peek` seemed to make people think it was a different request... (and `preview` seemed like the appropriate action). But I can rename it back, no problem.

(And this function uses `open_floating_preview` under the hood; what it does is take a `location`, extract the necessary information, and feed that to `open_floating_preview`. I thought that might be useful in other contexts as well -- keep functions small and composable.)